containers/jetty/src/main/java/com/yahoo/athenz/container/AthenzJettyContainer.java

```diff
@@ -469,6 +470,18 @@ void addHTTPSConnector(HttpConfiguration httpConfig, int httpsPort, boolean prox
             sslConnector.addBean(connectionLogger);
         }
         server.addConnector(sslConnector);
+
+        // Reload the key-store if the file is changed
+        final int reloadSslContextSeconds = Integer.parseInt(System.getProperty(AthenzConsts.ATHENZ_PROP_KEYSTORE_RELOAD_SEC, "0"));
+        if ((reloadSslContextSeconds > 0) && (sslContextFactory.getKeyStorePath() != null)) {
+            try {
+                KeyStoreScanner keystoreScanner = new KeyStoreScanner(sslContextFactory);
+                keystoreScanner.setScanInterval(reloadSslContextSeconds);
+                server.addBean(keystoreScanner);
+            } catch (IllegalArgumentException exception) {
+                LOG.error("Keystore will not be automatically reloaded when \"{}\" is changed: {}", sslContextFactory.getKeyStorePath(), exception.getMessage());
```

Comment:
I think that if reload is requested and there's an error during container boot we should throw an exception and prevent the service from coming up. Better than thinking it will be refreshed when it won't.

Reply:
Done